test/integration/app/spec_build.js

```diff
@@ -262,4 +262,43 @@ describe('Spec option and build integration tests for app generator', function (
     });
   });
 
+  describe('Web application with an object storage service is able to build', function () {
+    // Swift build is slow so we need to set a longer timeout for the test
+    this.timeout(300000);
+
+    var runContext;
+
+    var spec = {
+      appType: 'web',
```

Comment:
OK, I'm going to add checking into the refresh generator to validate the value of `appType`. I will do this in a separate PR, check the incorrect tests fail, then fix them.